test/api/searchResult.test.ts

```diff
@@ -0,0 +1,148 @@
+import { createMocks } from 'node-mocks-http';
+import { NextRequest, NextResponse } from 'next/server';
+
+import { GET } from '@/app/api/v1/searchResults/route';
+import { createManySearchResults } from '@/repositories/searchResult';
+import { createUser } from '@/repositories/user';
+
+import { user } from '../factories/user';
+import { searchResult } from '../factories/searchResult';
```

Comment:
Let's be more descriptive as I wondered where `searchResult` came from when reading the code below:

```suggestion
import { searchResultFactory } from '../factories/searchResult';
```

Reply:
Since, if I change this naming I have to change in all of the files that use it. Would it be good if I do this in the task where I move all the tests to one directory.

Reply:
Understood. Then ensure you keep track of these changes to make.